### PR TITLE
update aws-lambda-java-11 guide

### PR DESCRIPTION
**Pull request by sdelamo**
starter code has been updated so that by default you get an ApiGatewayProxyEvent with a Hello World output

### CODE REVIEW

**Review by burtbeckwith — APPROVED** [review]

File: guides/mn-application-aws-lambda-java11/mn-application-aws-lambda-java11.adoc [hunk 1]
```diff
@@ -6,29 +6,16 @@ common:completesolution.adoc[]
...
-source:Book[]
-
-* Annotate the class with `@Introspected` to generate the Bean Metainformation at compile time.
-
-It returns a `BookSaved` object:
-
-source:BookSaved[]
-
-* Annotate the class with `@Introspected` to generate the Bean Metainformation at compile time.
+* The `@Get` annotation maps HTTP requests to `/` to the `index` method.
```
Comment (burtbeckwith) [inline guides/mn-application-aws-lambda-java11/mn-application-aws-lambda-java11.adoc:14]: maybe use callout:get?

File: src/docs/common/snippets/common-lambda-test.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+  "path": "/",
+  "httpMethod": "GET",
+  "headers": {
+    "Accept": "application/json"
+  }
+}
+----
+
+When working with Amazon API Gateway is easy to use `apigateway-aws-proxy` as an `Event Template` to get started:
```
Comment (burtbeckwith) [inline src/docs/common/snippets/common-lambda-test.adoc]: "Gateway is easy" -> "Gateway, it's easy"

**Review by burtbeckwith — APPROVED** [review]

File: src/docs/common/snippets/common-lambda-test.adoc (hunk 2 above)
Comment (sdelamo) [reply src/docs/common/snippets/common-lambda-test.adoc]: Resolved via https://github.com/micronaut-projects/micronaut-guides/pull/949/commits/5fcd66d529e4890a039e003e1f95e0e4dcd0092f

File: guides/mn-application-aws-lambda-java11/mn-application-aws-lambda-java11.adoc (hunk 1 above)
Comment (sdelamo) [reply guides/mn-application-aws-lambda-java11/mn-application-aws-lambda-java11.adoc:14]: I think that we don't support callouts without `number` yet.

**Pull request merged**